README.md

```diff
@@ -45,7 +45,7 @@ $T(n) = 3^2T(\frac{n}{3^2}) + \frac{n^5}{3^4} + n^5$<br>
 Skipping the substitution step the next experession is:<br>
 $T(n) = 3^3T(\frac{n}{3^3}) + \frac{n^5}{3^5} + \frac{n^5}{3^4} + n^5$<br>
 
-Analysing the pattern, the expression for this recurrence relation is: $T(n) = 3^iT(\frac{n}{3^i}) + \displaystyle\sum{i=2}^k \frac{n^5}{3^{i+2}}  + n^5$<br>
+Analysing the pattern, the expression for this recurrence relation is: $T(n) = 3^iT(\frac{n}{3^i}) + \displaystyle\sum_{i=2}^k \frac{n^5}{3^{i+2}}  + n^5$<br>
```

Comment:
Can you please do one more step before skipping the substitution, for example, do it for T(n/81). Because I think what you end up as a pattern is not entirely correct!